mslice/plotting/plot_window/plot_figure.py

```diff
@@ -71,16 +74,23 @@ def add_slice_plotter(self, slice_plotter):
         self.actionGDOS.triggered.connect(partial(self.show_intensity_plot, self.actionGDOS,
                                                   self.slice_plotter.show_gdos, True))
 
-        self.actionHydrogen.triggered.connect(partial(self.toggle_recoil_line, self.actionHydrogen, 1))
-        self.actionDeuterium.triggered.connect(partial(self.toggle_recoil_line, self.actionDeuterium, 2))
-        self.actionHelium.triggered.connect(partial(self.toggle_recoil_line, self.actionHelium, 4))
+        self.actionHydrogen.triggered.connect(partial(self.toggle_overplot_line, self.actionHydrogen, 1, True))
+        self.actionDeuterium.triggered.connect(partial(self.toggle_overplot_line, self.actionDeuterium, 2, True))
+        self.actionHelium.triggered.connect(partial(self.toggle_overplot_line, self.actionHelium, 4, True))
         self.actionArbitrary_nuclei.triggered.connect(self.arbitrary_recoil_line)
-
-    def toggle_recoil_line(self, action, relative_mass):
+        self.actionAluminium.triggered.connect(partial(self.toggle_overplot_line, self.actionAluminium,
+                                                       'Aluminium', False))
+        self.actionCopper.triggered.connect(partial(self.toggle_overplot_line, self.actionCopper, 'Copper', False))
+        self.actionNiobium.triggered.connect(partial(self.toggle_overplot_line, self.actionNiobium, 'Niobium', False))
+        self.actionTantalum.triggered.connect(partial(self.toggle_overplot_line, self.actionTantalum,
+                                                      'Tantalum', False))
```

Comment:
For me for some reason `partial` passes `True` for the last argument (`extra_info`) instead of leaving as the default (`None`) - so the `Aluminium` etc lines gives me an error (since it tries to find a CIF file called `True`. So I had to add another `False` to the end of these lines...

Reply:
Apparently the action also passes in the checkbox status as True/False so this was getting confused for the default parameter. Should be ok now.